config/default.yml

```diff
@@ -92,9 +92,11 @@ linters:
     enabled: true
     style: exclude_zero # or 'include_zero'
 
-  MergeableSelector:
+  MergeEqualSelector:
```

Comment:
I think a better name in this case is `MergeEquivalentSelector`, even if perhaps more verbose.